### ISSUE TITLE
Zooming doesn't work in Safari browser

### ISSUE THREAD

**Issue by SerovMihail** [ValentinH/svelte-easy-crop#35]
## Are you reporting a bug?

Basic example is relevant and bug could be reproduced -> https://codesandbox.io/s/svelte-easy-crop-basic-demo-q1005?file=/App.svelte. Also reproduces on react-easy-crop basic example.

### Env: 

- Safari version -> Version 15.6.1 (17613.3.9.1.16)
- macOS Montrey 12.5.1 (21G83)

### Provide the steps to reproduce the issue, e.g.:

- Open sandbox
- Try to pinch by two fingers. I'm using a laptop touchpad for it 

 ### Expected behaviour 
 Pinch works correctly in Safari

**Comment by ValentinH** [comment]
Does it work on another macOs browser?

**Comment by SerovMihail** [comment]
Yes, I asked my colleague, he uses macos catalina and safari version is 14.0. Bug reproduced.

**Comment by ValentinH** [comment]
I meant on a non-safari browser on macOS like Chrome. I tried it kinda work even though it doesn't feel natural.

I don't know if there is a way to listen to this gesture on safari.

**Comment by ValentinH** [comment]
Ok this is probably this one that should be handled: https://developer.mozilla.org/en-US/docs/Web/API/Element/gesturechange_event

Today, we simply prevent the default.

**Comment by ValentinH** [comment]
I fixed this in the react version a few minutes ago: https://github.com/ValentinH/react-easy-crop/pull/414

Would you be interested in adding the same to this svelte version? I haven't touched Svelte for years 🙈

**Comment by SerovMihail** [comment]
Don't you think that the math of the zooming algorithm is wrong? I don't know why but all versions higher than 4.6.0 can zoom but the destination point is wrong; I mean i get stuck in corner every time i try to zoom center of the image, no matter where and how I pitch on my touchpad zooming always behave like this.

Attaching video to show how it works 

https://user-images.githubusercontent.com/22121721/195927809-cb7d0705-095e-4b64-98b8-982d7894e693.mov

**Comment by ValentinH** [comment]
I actually noticed this as well but  it's only on codesandbox and using safari if I'm not mistaken. Therefore, it's probably due to iframes and the gesture client X/Y not being correct. I'll have to investigate...

So you confirm your video is taken on Safari?

**Comment by SerovMihail** [comment]
> So you confirm your video is taken on Safari?
Yes, the video was taken from safari

**Comment by JE4GLE** [comment]
For me it is as well that [multi-touch zoom gestures](https://support.apple.com/en-us/102482#:~:text=Zoom%20in%20or%20out,-Pinch%20with%20two) are not correctly working. I'm using Safari with macOS 26.0.1
